### ISSUE TITLE
Unhardcode the chat actions

### ISSUE THREAD

**Issue by Hazy-uhyR** [beyond-all-reason/Beyond-All-Reason#2579]
### Description

Currently, most of the chat stuff is hardcoded, so as far as I can tell, the various keyset+action combos used by the hardcoded chat actions can not be changed with custom keybinds.

If you do do keybind stuff like the chatally action, you also don't get the neat box, but rather (in the case of chatally) the big "Say: a:" prompt.

### Why?

It would be nice if this stuff was unhardcoded so that we can further customize the game via uikeys.txt.

### What area of the game does this feature request pertain?

Settings, Controls or Hotkeys

**Comment by hihoman23** [comment]
What do you mean with most of the stuff?
There is a lot to keybind which nobody would ever touch(like ctrl+v, etc.)

**Comment by Hazy-uhyR** [comment]
You're right, I have described this poorly. I mean alt-enter, enter, ctrl-enter, those keys are hardcoded to always be chat functions, and cannot be changed (afaik).

Further, if you try to bind those functions (such as chatally action), you don't get the neat box, but rather the big "Say: " promt, with it being "Say: a:" in the case of sending a chatally action via keybinds.

**Comment by hihoman23** [comment]
Closing in favor of #5756
